applications/ink-explorer.md

```diff
@@ -0,0 +1,189 @@
+| 0e. | Article | We will post an article on Twitter and Reddit for both english and spanish speakers communities.
+| 1. | Create database | Create a DB on PostgreSQL to store all the information |  
+| 2. | Backend service | Create a service that subscribes to blockchain events and Ink modules events. This service will store and index the information in the database to be accessed quickly and easily. |  
+| 3. | API | Extend the service functionality to expose an API that gets the contracts data |
+| 4. | Testing | Achieve a testing coverage of the functionalities above 90% |
+
+Deliverables in this milestone will be dockerized. We will provide a README with examples showing how to start the service, and query the API.
+
+#### 1 - Create database
+
+Create a docker container to start a PostgreSQL database.
+Define the models to store.
+Create tables and indexes.
+
+#### 2 - Backend service
+
+Create a docker container to start a typescript project.
+Add all subscriptions needed to retrieve the information of the emited events from both the blockchain (using polkadot.js) and Ink modules. 
+Store and index the events data on the database. (If needed sync all data from blockHash - can be from genesis but it will take longer)
+
+#### 3 - API
+
+Create an interface to get all the contracts information from the database.
+
+#### 4 - Testing
+
+Add unit tests and end to end test to achieve a high coverage of the functionality.
+
+### Milestone 2 - Frontend
+
+- **Estimated Duration:** 1.5 month
+- **FTE:**  2
+- **Costs:** 15,000 USD
+
+This milestone is entirely about creating a frontend app and making it work along with the back end service. The app will be mounted on the same docker as the backend service so it can be started with a single docker command.
+
+| Number | Deliverable | Specification | 
+| ------------- | ------------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will provide both **english** and **spanish** versions of the documentation. This will cover step by step how to set up and start the project using both docker and running the node locally. |
+| 0c. | Testing Guide | Unit test and end to end tests will cover the core functions to ensure everything works as expected. The documentation will have an example on how to run these tests. |
+| 0d. | Docker | A Dockerfile will be provided that will be able to start the node and run tests for all the functionality delivered within this milestone. |
+| 0e. | Article | We will post an article on Twitter and Reddit for both english and spanish speakers communities.
+| 1. | Design | Design the application frontend |  
+| 2. | Create App | Create a frontend application that mocks the data |  
+| 3. | Interact with backend | Integrate both frontend and backend replacing the mocks with real data from the service |  
```

Comment:
D2.1 is basically what you provided in your mockup, and since you are submitting the frontend in this milestone as well, we don't really need the design. Feel free to remove this deliverable. 

D2.2 uses mock data, but D2.3 uses real data. I'm guessing this just reflects your internal roadmap, but it's an odd qualification here. Instead, it'd be great if you could list the functionalities that are included in the frontend, so we get a better understanding of the extent of the deliverable. It also helps us evaluate the milestone quicker.

Reply:
@semuelle I pushed some changes :+1: . Let me know if we need to change anything else. Thanks!